### PR TITLE
Metagraph Explorer

### PR DESCRIPTION
**Pull request by paul-tqh-nguyen**
This PR is for the Metagraph explorer.

### CODE REVIEW

File: metagraph/core/planning.py [hunk 1]
```diff
@@ -197,12 +207,7 @@ def __repr__(self):
                 else:
                     s.append(f"** {varname} **")
                     anni = sig.parameters[varname].annotation
-                    if type(anni) is Wrapper:
-                        s.append(f"{anni.__name__}")
-                    elif type(anni) is type:
-                        s.append(f"{anni.__name__}")
-                    else:
-                        s.append(f"{anni.__class__.__name__}")
+                    s.append(self.__class__.string_for_annotation(anni))
```
Comment (jim22k) [inline metagraph/core/planning.py:210]: `self` has access to all classmethods and staticmethods. No need to use `self.__class__` here.

File: docs/getting_started/tutorial.ipynb [hunk 2]
```diff
@@ -18,7 +18,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 1,
```
Comment (jim22k) [inline docs/getting_started/tutorial.ipynb:21]: Be careful about pushing changes to a notebook which aren't substantive changes. I always try to "Restart kernel and clear all outputs" before saving the notebook. That usually avoids execution_count changes and such.

**Review by jim22k — APPROVED** [review]

File: docs/getting_started/tutorial.ipynb (hunk 2 above)
Comment (paul-tqh-nguyen) [reply docs/getting_started/tutorial.ipynb]: https://github.com/metagraph-dev/metagraph/pull/120/commits/d303ca095a4410154834df02e3de7e9bb871da3e removes these accidental changes.

File: metagraph/core/planning.py (hunk 1 above)
Comment (paul-tqh-nguyen) [reply metagraph/core/planning.py]: https://github.com/metagraph-dev/metagraph/pull/120/commits/3022582fd357e1d6cd330caa32cd984df3d0fc6f removes `self.__class__`.

**Pull request merged**